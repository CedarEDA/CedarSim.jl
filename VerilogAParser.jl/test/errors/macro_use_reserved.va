module macro_use_reserved(x);

analog function foo;
    foo = TYPE == `nmos ? 7.45669e11 : 1.16645e12;
endfunction

endmodule