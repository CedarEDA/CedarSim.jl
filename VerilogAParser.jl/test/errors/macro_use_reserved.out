[31m[1mâ•­ ERROR [VAS03]:[22m[39m[31m Use of backtick not followed by a macro name.[39m
[31mâ”‚[39m[90m  â•­â”€â”€â”€â”€â”€ macro_use_reserved.va:4[39m
[31mâ”‚[90m4[0m [90mâ”‚[0m     foo = TYPE == [91m`[0mnmos ? 7.45669e11 : 1.16645e12;
[31mâ”‚[0m  [90mâ”Š[0m                   [91mâ”“
[31mâ”‚[0m  [90mâ”Š[0m          [91mMissing macro name[0m
[31mâ”‚[39m[90m  â”‚[39m
[31mâ”‚[39m[90m  â•°â”€â”€â”€â”€â”€ [39mUse of backtick not followed by a macro name.
[31mâ”‚[39m
[31mâ”‚[39m[34m help: [39mThe parser expected a macroname after this backtick, but instead found [0m[1ma reserved word[22m.
[31mâ”‚[39m       Check for spelling mistakes.
[31mâ•¯[39m
